### DISCUSSION TITLE
How can I get the color of a pixel on image

### DISCUSSION THREAD

**Discussion by Decodetalkers** [image-rs/image#1850] [Q&A] [answered]
I want to get rgb code with the position of the image

**Comment by fintelia** [comment, marked as answer]
What have you tried? The details will depend on what type of image you are working with, but for instance ImageBuffer has a [`get_pixel` method](https://docs.rs/image/latest/image/struct.ImageBuffer.html#method.get_pixel).

**Reply by Decodetalkers** [reply]
> What have you tried? The details will depend on what type of image you are working with, but for instance ImageBuffer has a [`get_pixel` method](https://docs.rs/image/latest/image/struct.ImageBuffer.html#method.get_pixel).

Thanks
